package.json

```diff
@@ -154,14 +159,13 @@
     "linters": {
       "*.{js,json,css}": [
         "prettier --write",
-        "eslint --fix --max-warnings 0",
+        "eslint --fix --max-warnings 2",
```

Comment:
eslint in their infinite wisdom warn about ignored files - https://github.com/eslint/eslint/issues/5623.  Therefore have to set max warnings higher in order to get past it.